app/models/user.rb

```diff
@@ -57,6 +57,16 @@ class User < ApplicationRecord
 
   scope :with_same_userid, ->(id) { where(:userid => User.where(:id => id).pluck(:userid)) }
 
+  LOOKUP_SCOPES = [:api_includes].inject(Hash.new { :default_scoped }) {|h,i| h[i] = i; h}.freeze
```

Comment:
could you put a comment above this stating what we are storing in here.
looks like [scope][userid] and it stores a scope.

most of these seem like they will just hold default_scope - would it make sense to not have a hash with a default but instead use a `||`? wondering how many fewer objects we would produce

Reply:
It actually is just the following:

```
{ :api_includes => :api_includes }
```

And then for anything else, it just returns `:default_scoped`.  It is just a lookup table for what methods are allowed, and otherwise defaults to `.send(:default_scoped)`.